Model/ConfigProvider/Idin.php

```diff
@@ -138,6 +142,10 @@ protected function isIDINActive()
                 $customer              = $this->customerRepositoryInterface->getById($customerId);
                 $customerAttributeData = $customer->__toArray();
                 $active                = (isset($customerAttributeData['custom_attributes']) && isset($customerAttributeData['custom_attributes']['buckaroo_idin']) && $customerAttributeData['custom_attributes']['buckaroo_idin']['value']) ? false : 1;
+            }elseif($active === true){
```

Comment:
minor thing, but could you please follow phpcs to eliminate such things (e.g.: wrong space indentation)

Reply:
sorry, fixed